### PR TITLE
I added a new translation (basque, "eu") and translated 2 lines of span…

### PR DESCRIPTION
**Pull request by cesetxeberria**
I added a new translation (basque, "eu") and translated 2 lines of spanish ("es") which were still in english

### CODE REVIEW

File: lang/eu.inc.php [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# GPL License: http://www.gnu.org/licenses/gpl.txt
+#
+#==============================================================================
+
+#==============================================================================
+# Spanish
```
Comment (coudot) [inline lang/eu.inc.php]:
```suggestion
# Basque
```

Comment (coudot) [reply lang/eu.inc.php]: @cesetxeberria could you just change this?

Comment (cesetxeberria) [reply lang/eu.inc.php]: Thanks, didn't seen it

**Comment by coudot** [comment]
Thanks!

**Pull request merged**